packages/teams-js/src/public/runtime.ts

```diff
@@ -263,6 +263,8 @@ export const teamsRuntimeConfig: Runtime = {
     },
     interactive: {},
     logs: {},
+    location: {},
```

Comment:
Just to verify: all of these capabilities are fully supported in Teams v1. Is that correct?

Reply:
Just the location :)